test/svg-sprite.test.js

```diff
@@ -38,6 +37,11 @@ const expectationsPath = require('./helpers/expectations-path.js');
 
 const dest = tmpPath;
 
+// This is needed so that we don't hit the `fs.rmdir`
+// deprecation warnings on Node.js >= 14.14.0
+// TODO Drop this when we drop support for Node.js 12
+const rm = fs.promises.rm ? fs.promises.rm : fs.promises.rmdir;
```

Comment:
This can be simplified to `fs.promises.rm || fs.promises.rmdir;`

Reply:
I tend to prefer the plain old ternary since we can't use `??` yet, but both achieve the same. Generally, nowadays, I don't like using short and seemingly "smarter" code, but rather more explicit :)